app/uk/gov/nationalarchives/omega/editorial/controllers/EditSetController.scala

```diff
@@ -296,14 +269,47 @@ class EditSetController @Inject() (
     }
   }
 
+  private def prepareForDisplay(originalEditSetRecord: EditSetRecord): EditSetRecord =
+    Seq[RecordTransformer](prepareCreatorIDs, preparePlaceOfDeposit).foldLeft(originalEditSetRecord)(
+      (editSetRecord, transformer) => transformer(editSetRecord)
+    )
+
+  private def prepareCreatorIDs(editSetRecord: EditSetRecord): EditSetRecord = {
+    val recognisedCreatorIds = editSetRecord.creatorIDs.filter(isCreatorRecognised)
+    val updatedCreatorIds = if (recognisedCreatorIds.nonEmpty) recognisedCreatorIds else List.empty
```

Comment:
Does this expression do anything now? Won't it always evaluate to `recognisedCreatorIds` or the equivalent `List.empty`?

Reply:
You're correct. I'll change.

Reply:
Changed.